examples/windowing/noWindowExample/README.md

```diff
@@ -0,0 +1,39 @@
+#About noWindowExample
+
+
+###Learning Objectives
+
+This example demonstrates how to use openframeworks to run processes without a window. 
+
+In the code, pay attention to: 
+
+* Use of ````	ofAppNoWindow window```` in the main.cpp.
+* The fact that no draw commands will be displayed. See ````draw()````
+
+
+### Expected Behavior
+
+When launching this application you will see the following:
+
+* Openframeworks has no window.
+* CommandLine is running as expected.
+
+
+
+When the key ````r```` is pressed:
```

Comment:
Instead of giving examples that won't produce a result, just that there is no (current) way to accept command line or mouse input with no-window mode.